package.json

```diff
@@ -26,6 +26,7 @@
     "release": "release-it"
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
```

Comment:
Please remove this, shouldn't be necessary.

Reply:
I was having trouble getting it to link inside of my project.